python/idsse_common/idsse/common/vectaster.py

```diff
@@ -390,12 +426,20 @@ def _pixels_for_polygon(
     return pixels
 
 
-def _round_x_y(x, y, rounding: Union[str, RoundingMethod]) -> Pixel:
+def _is_coord(arg) -> bool:
+    return isinstance(arg, Iterable) and all(isinstance(v, Number) for v in arg)
+
+
+def _is_coords(arg) -> bool:
+    return isinstance(arg, Iterable) and all(_is_coord(v) for v in arg)
+
+
+def _round(*args, rounding: str | RoundingMethod) -> list[int]:
     if isinstance(rounding, str):
         rounding = RoundingMethod[rounding.upper()]
 
     if rounding is RoundingMethod.ROUND:
-        return (round_(x), round_(y))
+        return [round_half_away(v) for v in args]
```

Comment:
I'm just thinking about my refactor task to consolidate functions like this to once place.

Should we consider renaming `round_half_away` to something less verbose, like `round_`? Or maybe `round_()` is a separate function that wraps `round_half_away()` with this round-or-floor logic? I assume we don't plan to use the built-in `round` anywhere in IDSSe, so round_half_away is a bit overly descriptive with the current name.